SharperUniverse/Persistence/SharperSaveModel.cs

```diff
@@ -4,11 +4,25 @@
 
 namespace SharperUniverse.Persistence
 {
+	
+	/// <summary>
+	/// A save model to represent the current game state, used by the <see cref="PersistenceManager"/>.
+	/// </summary>
 	public class SharperSaveModel
 	{
+		/// <summary>
+		/// The Id of the current save.
+		/// </summary>
 		public int Id { get; set; }
```

Comment:
Is this mutable because of LiteDB?

Reply:
Yes - and for other databases that might need it, too.